.gitignore

```diff
@@ -0,0 +1,2 @@
+.ipynb_checkpoints
+*/.ipynb_checkpoints/*
```

Comment:
The #2 line can be removed. The first line already ignores all checkpoints recursively for the repo.